server/lib/pbench/test/fixtures/upload/bad_tarball/bad_log.tar.xz.md5

```diff
@@ -0,0 +1 @@
+9d5a479f6f75fa9b3bab27ef79ad5b29
```

Comment:
No, this doesn't get removed at the end of the completion of process_tb process. This is a failure situation where .md is present but its respective tar.xz is not present. So, I didn't need to copy it again.

Reply:
I don't understand...what files are supposed to be under `server/lib/pbench/test/test-fixtures` and what files are supposed to be under `server/lib/pbench/test/fixtures`?  (The directory names are too similar to each other for me to guess what the differences are between them.)

These both appear to be paths in the repo, so the files in them should be tracked by Git, and they should not be test _outputs_....

Reply:
`test-fixtures` directory consists of `tarball` and its .md5 file. This directory acts as a backup from where I can fetch tarball and its .md5 file again after a test run is complete.

'fixtures' is the directory where test looks for tarballs and process them.

Reply:
So, `test-fixtures` is where we keep artifacts (files) used in testing, and they are tracked by Git.  Cool.

Meanwhile, `fixtures` is a "working directory" where the unit test framework places copies of the artifacts, so that the CUT doesn't damage the originals.  This makes good sense, except for the fact that, currently, the directory is in the middle of the check-out of the repository.  This directory should be a temporary directory which is created wherever the test system creates temporary directories and it should be destroyed when the tests are finished.

And, `server/lib/pbench/test/fixtures/upload/bad_tarball/bad_log.tar.xz.md5` should live in `test-fixtures`, and the test system should copy it to wherever it is needed when the pertinent test is run.